devices/Rtc/README.md

```diff
@@ -1,25 +1,55 @@
 # Realtime Clock
-The RTC devices supported by the project include DS1307, DS3231, PCF8563.
+The RTC devices supported by the project include DS1307, DS3231, PCF8563 (or compatible BM8563).
+
+## Documentation
+- DS1307 [datasheet](https://cdn.datasheetspdf.com/pdf-down/D/S/1/DS1307-Maxim.pdf)
+- DS3231 [datasheet](https://datasheets.maximintegrated.com/en/ds/DS3231.pdf)
+- PCF8563 / BM8563 [datasheet](https://cdn.datasheetspdf.com/pdf-down/P/C/F/PCF-856.pdf)
+
+## Board
+
+![Circuit RTC](circuit_rtc.png)
 
 ## Usage
+
+### Hardware Required
+
+- DS1307/DS3231/PCF8563/BM8563
+- Male/Female Jumper Wires
+ 
 ```C#
+using System.Device.I2c;
 using Iot.Device.Rtc;
+using nanoFramework.Hardware.Esp32;
+
+// The I2C pins 21 and 22 in the sample below are MCU specific
```

Comment:
rather than MCU, use ESP32 as the Configuration.SetPinFunction is ESP32 specific.
So add a word that this can be different for the other support MCU

Reply:
done, thanks!